crates/linker/src/pe.rs

```diff
@@ -48,7 +49,18 @@ struct PeMetadata {
     dynamic_relocations: DynamicRelocationsPe,
 
     /// File offset for the thunks of our dummy .dll
-    thunks_start_offset: usize,
+    thunks_start_offset_in_file: usize,
```

Comment:
super nit: you might want these to be U64 since this gets serialized to a file. If the compiler is built on a 32 bit system but cross compiles to a 64 bit system, that could matter.

Reply:
that won't help: we still index into the file with a `usize` (that is what rust kind of forces on you) so an executable bigger than 4Gb will just fail in all sorts of places on a 32-bit system.